scripts/runfabtests.sh

```diff
@@ -173,7 +175,7 @@ function unit_test {
 	local end_time
 	local test_time
 
-	local e=$(is_excluded $test)
+	local e=$(is_excluded $test_exe)
```

Comment:
Ick.  That level of subtlety (deliberately allowing `$test_exe` to expand and be word-split at invocation time so that only `fi_foo` ends up in `$1`) warrants a comment or a change in the logic to make it clear.  More obvious would be to pass `"fi_${test}"` instead of `$test_exe`.